content/projects/coursera/it-automation/course-1-python-crash-course/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+_db_id:
```

Comment:
remove `_db_id:` completely from every new file, tilde will add these in when it consumes the syllabus